data/fleets.txt

```diff
@@ -1191,6 +1191,19 @@ fleet "Large Republic"
 		"Frigate" 2
 		"Rainmaker"
 		"Gunboat"
+	variant 4
```

Comment:
Rather than modify the core `Large Republic` fleets with these variants, consider creating new fleets (`Navy Cargo / Navy Transport / Navy Support`) in which the Auxiliary and Dropship feature prominently. Then we can specifically use navy fleets that are not combat intensive when desired, and know that the (many) existing uses of `Small Republic` and `Large Republic` (and subsequent modifications by events or plugins) are not impacted.

e.g.
```
system Alnitak
    ....
    fleet "Large Navy Support" 8000
    fleet "Small Navy Support" 4000
    ....
```

Reply:
yeah, i had planned on doing that eventually, but put them there for now to get the fleets recorded and able to be adjusted